### ISSUE TITLE
Accessibility: Collection search field has no label

### ISSUE THREAD

**Issue by flutterflies** [uclibs/scholar_uc#1690]
### Descriptive summary

The search bar on the collection view page has no text label which makes screen readers unable to inform users what the form element is for
